sycl/include/sycl/detail/property_helper.hpp

```diff
@@ -33,6 +33,8 @@ enum DataLessPropKind {
   UseDefaultStream = 8,
   DiscardEvents = 9,
   DeviceReadOnly = 10,
+  // TODO(#15184): Remove the following fusion-related entries in the next
```

Comment:
#15185?

Reply:
No, I've created an issue to track this in https://github.com/intel/llvm/issues/15184.

Reply:
Perfect!